Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -150,11 +136,21 @@ - (instancetype)initWithTextView:(UITextView<RCTBackedTextInputViewProtocol> *)b
   if (self = [super init]) {
     _backedTextInputView = backedTextInputView;
     backedTextInputView.delegate = self;
+    backedTextInputView.textStorage.delegate = self;
   }
 
   return self;
 }
 
+#pragma mark -
+- (void)textStorage:(NSTextStorage *)textStorage
+  didProcessEditing:(__unused NSTextStorageEditActions)editedMask
+              range:(__unused NSRange)editedRange
+     changeInLength:(__unused NSInteger)delta {
+  [_backedTextInputView.textInputDelegate textInputDidChange];
```

Comment:
I am worried a bit about over-firing. So, e.g. if only text attributes were changed, we don't want to send an event.

Reply:
I will check if this is the case & fix it if that does occur.